README.md

```diff
@@ -34,7 +37,14 @@ config = Config(
 vrt = VisualRegressionTracker(config)
 ```
 
-### Send image
+### Setup
+
+```python
+vrt.start();
```

Comment:
This example could be made more "pythonic" if context manager support was added.
The code could become:
`with vrt:
    vrt.track(...)`

I'm happy to add the change if you would like.

Reply:
sure, please, change it